jenkins/Dockerfile

```diff
@@ -32,20 +32,21 @@ wget https://aka.ms/downloadazcopy-v10-linux -O /opt/azurecopy.tar.gz && tar xzv
 ENV GOROOT /usr/local/go
 ENV GOBIN  /usr/local/go/bin
 ENV GOPATH /usr/local/go/src
-ENV PATH $GOROOT:$GOBIN:$GOPATH:$PATH
+ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64/
+ENV PATH $GOROOT:$GOBIN:$GOPATH:$JAVA_HOME:$PATH
+
 
 COPY jenkins/plugins.txt /usr/share/jenkins/ref/plugins.txt
 RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh && /usr/local/bin/install-plugins.sh < /usr/share/jenkins/ref/plugins.txt
 
 ADD jenkins/files/xmls.tar /var/jenkins_home/
-COPY --chown=jenkins:jenkins jenkins/files/encrypt.groovy /var/jenkins_home/
-COPY --chown=jenkins:jenkins jenkins/files/credentials.xml /var/jenkins_home/
-COPY --chown=jenkins:jenkins jenkins/files/nodeMonitors.xml /var/jenkins_home/
-COPY --chown=jenkins:jenkins jenkins/files/config.xml /var/jenkins_home/
-COPY --chown=jenkins:jenkins jenkins/files/scriptApproval.xml /var/jenkins_home/
+COPY  jenkins/files/copy_reference_file.log /usr/share/jenkins/ref/copy_reference_file.log
+COPY  jenkins/files/encrypt.groovy /usr/share/jenkins/ref/encrypt.groovy
```

Comment:
Why are we changing the following 5 lines.?

Reply:
During the ECS - EFS integration the Jenkins image was running into issues during spin up. So I had to make these changes to work.